components/do_verify_email/index.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {createSelector} from 'reselect';
+import {verifyUserEmail} from 'mattermost-redux/actions/users';
+
+import DoVerifyEmail from './do_verify_email.jsx';
+
+const mapStateToProps = createSelector(
```

Comment:
Is the selector really needed here? Also it's not returning any props so won't you be losing the `location` prop? Let me know if I'm missing something

Reply:
You're right about the selector not being needed. What I was wondering about this particular component though, is that it seems to only be called through a link in a verification mail, rather than by a parent component (likely then getting its `location` prop through parameters given in the link).

It seems like the `mapStateToProps` function is required by `connect()`, so I wasn't sure what was the best course of action for a component that only needed the `mapDispatchToProps`. 
Would the following be the right way to do it then?
```
function mapStateToProps(state, ownProps) {
    return ownProps;
}
```